server/core/store.go

```diff
@@ -251,9 +251,9 @@ func (s *StoreInfo) GetLastHeartbeatTS() time.Time {
 const minWeight = 1e-6
 const maxScore = 1024 * 1024 * 1024
 
-// LeaderScore returns the store's leader score: leaderSize / leaderWeight.
-func (s *StoreInfo) LeaderScore(delta int64) float64 {
-	return float64(s.GetLeaderSize()+delta) / math.Max(s.GetLeaderWeight(), minWeight)
+// LeaderScore returns the store's leader score: leaderCount.
```

Comment:
Why not `leaderCount / leaderWeight`?

Reply:
And I think it will be good idea to consider `delta`.

Reply:
I'm not sure that it is associated between leader size and leader weight.
Before delta is according to leader region size, so we need a better method to get delta, but I have no good idea.

Reply:
```
// ResourceSize returns delta size of leader/region by influence.
func (s StoreInfluence) ResourceSize(kind core.ResourceKind) int64 {
	switch kind {
	case core.LeaderKind:
		return s.LeaderSize
	case core.RegionKind:
		return s.RegionSize
	default:
		return 0
	}
}
```
I think change `s.LeaderSize` to `s.LeaderCount` will perfectly do the job. cc @rleungx

Reply:
LGTM. But when balancing leader , need we consider the tolerant ratio ?

Reply:
I think we need it. You can treat the ratio as region count too.

Reply:
PTAL